components/DiscoverFeed/DiscoverFeed.tsx

```diff
@@ -80,7 +83,7 @@ export default function DiscoverFeed() {
             itemCount={events.length}
             itemSize={getRowHeight}
             width={width}
-            overscanCount={10}
+            overscanCount={5}
```

Comment:
Reduced these to 5 to reduce the amount of subscriptions being made to relays. Looks better now IMO.